sparks_pif_converters/XRD/TOPAS_refinement_to_pif.py

```diff
@@ -0,0 +1,83 @@
+import csv
+import argparse
+from pypif import pif
+from pypif.obj import *
+import os
+
+
+def parse_TOPAS_refinement(TOPAS_refinement_file):
+    '''
+    Function that converts the output file from a TOPAS XRD refinement to a PIF.
+
+    Args:
+        TOPAS_refinement_file (csv or csv-formatted txt): The output file from a refinement.
+
+    Returns:
+        pif (object): A pif object containing exp, calc, and diff curves as properties.
+    '''
+
+    # Initialize arrays to hold values
+    theta = []
+    exp = []
+    calc = []
+    diff = []
+    phase1 = []
+    phase2 = []
+
+    # read csv, iterate through rows
+    reader = csv.reader(open(TOPAS_refinement_file, 'rb'))
+    for index,  row in enumerate(reader):
+
+        # append values
+        if index > 1:
+            theta.append(row[0])
+            exp.append(row[1])
+            calc.append(row[4])
+            diff.append(row[5])
+            phase1.append(row[6])
+            phase2.append(row[7])
+
+    # create system
+    my_pif = ChemicalSystem()
+    my_pif.names = [os.path.basename(f).rpartition(".")[0]]
+    my_pif.properties = []
+
+    # create properties
+    p_exp = Property(name="Counts", scalars=exp, units="arbitrary units")
+    p_exp.conditions = [Value(name="2$\\theta$", scalars=theta, units="$^\circ$"), Value(name="Data Type", scalars="")]
```

Comment:
I added an extra condition to differentiate between the curves since they will be plotted together. This is just a placeholder for now while site plotting functionality is being developed.